addon/components/horizontal-bar-chart.js

```diff
@@ -237,13 +237,24 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     };
   }),
 
+  /*
+   * Ensures that grouping labels will be on the left when all data is negative or 0
+   * @private
+   * @param {Object}
+   * @return {Boolean}
+   */
+  _isNotPositive: function(d) {
```

Comment:
Yeah I was having trouble coming up with a good name for that function, will change it